Realm/RLMRealm.mm

```diff
@@ -70,6 +70,13 @@ - (void)stop {
     _block = nil;
 }
 
+- (void)suppressNextNotification {
+    auto block = _block;
```

Comment:
A brief comment or different variable name might help explain what this is doing.

It wasn't immediately clear to me that this approach to suppressing notifications completely bypasses the Object Store's mechanism for suppressing notifications.